From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Tue, 15 Nov 2016 21:16:50 +0100
Subject: remove external inclusion of modernizr.min.js

We don't want to load minimized JS from the net, we have a local
installed package libjs-modernizr there we can use the JS file from.
---
 doc/source/_theme/layout.html                  | 2 +-
 doc/source/_theme/sphinx_rtd_theme/layout.html | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/source/_theme/layout.html b/doc/source/_theme/layout.html
index d7a59e1..aff837f 100644
--- a/doc/source/_theme/layout.html
+++ b/doc/source/_theme/layout.html
@@ -94,7 +94,7 @@
   {%- endblock %}
   {%- block extrahead %} {% endblock %}
 
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.6.2/modernizr.min.js"></script>
+  <script src="/usr/share/javascript/modernizr/modernizr.js"></script>
 
 </head>
 
diff --git a/doc/source/_theme/sphinx_rtd_theme/layout.html b/doc/source/_theme/sphinx_rtd_theme/layout.html
index d7a59e1..aff837f 100644
--- a/doc/source/_theme/sphinx_rtd_theme/layout.html
+++ b/doc/source/_theme/sphinx_rtd_theme/layout.html
@@ -94,7 +94,7 @@
   {%- endblock %}
   {%- block extrahead %} {% endblock %}
 
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.6.2/modernizr.min.js"></script>
+  <script src="/usr/share/javascript/modernizr/modernizr.js"></script>
 
 </head>
 
